docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -4,6 +4,70 @@
 Diarrheal Diseases
 ==================
 
+Disease Description
+-------------------
+
+.. todo::
+
+   Adapt material from GBD capstone and other sources, e.g. [WHO]_,
+   [CDC]_, [Wikipedia]_, [GBD-2017-YLD-Capstone-Appendix-1]_
+
+The relevant ICD codes are...
+
+Modeling Diarrheal Diseases in GBD 2017
+---------------------------------------
+
+.. todo::
+
+   Add relevant detail about measles modeling process from
```

Comment:
Measles?

Reply:
```suggestion
   Add relevant detail about diarrheal diseases modeling process from
```

Reply:
woah, cool, i've got to learn how to use suggested changes like this.